common/thorhelper/roxiehelper.cpp

```diff
@@ -1666,7 +1666,7 @@ void CSafeSocket::sendSoapException(IException *E, const char *queryName)
     {
         StringBuffer error("While reporting exception: ");
         EE->errorMessage(error);
-        DBGLOG("%s", error.str());
+        OERRLOG("%s", error.str());
```

Comment:
likewise not sure this is operator - probably programmer.  I suspect there should be quite a high threshold on something going to the operator - misconfigured system, fatal errors, kernel errors.